.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   test:
     name: Test ${{ matrix.os }} Python ${{ matrix.python-version }} conda=${{ matrix.use-conda }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 12
```

Comment:
I was on my trip when you merged this earlier, but something like `15` minutes might be safer—particularly with the macOS builds, I've seen it sometimes stall for 5 or perhaps even 10 minutes on one step and then recover and succeed, mostly driven (as far as I can tell) by slowness and non-determinism in the conda solver.